### DISCUSSION TITLE
Multiple select with options populated with x-for

### DISCUSSION THREAD

**Discussion by fabioflebus** [alpinejs/alpine#4258] [1. Help]
Hi,
I've created the following multiple select

```html
<div x-data="{sel:['a','b','x'], list:[{id:'x',name:'X'},{id:'y',name:'Y'}]}">
    <select x-model="sel" multiple style="height:100px; width: 200px">
      <option value="a">A</option>
      <option value="b">B</option>
      <option value="c">C</option>
      <template x-for="(l) in list" :key="l.id">
        <option :value="l.id" x-text="l.name"></option>
      </template>
    </select>
</div>
```

I'm expecting that **a,b,x** options are selected, but it seems that options created with x-for are not bound to the x-model.

Thank you

![screen](https://github.com/alpinejs/alpine/assets/139539981/25dc0e0b-fd02-49ef-abdc-c03767f57953)

**Comment by SimoTod** [comment]
See https://github.com/alpinejs/alpine/discussions/1872#discussioncomment-1084095
